summary/views.py

```diff
@@ -310,58 +354,142 @@ def get_scenario_data(data, run_uuid):
 
                     # Year 1 Savings
-                    year_one_costs = tariff.get('year_one_bill_us_dollars') or 0
-
-                    year_one_costs_bau = tariff.get('year_one_bill_bau_us_dollars') or 0
-
-                    results['year_one_savings_us_dollars'] = year_one_costs_bau - year_one_costs
-
+                    year_one_costs = sum(filter(None, [
+                        tariff.get('year_one_energy_cost_us_dollars') or 0,
+                        tariff.get('year_one_demand_cost_us_dollars') or 0,
+                        tariff.get('year_one_fixed_cost_us_dollars') or 0,
+                        tariff.get('year_one_min_charge_adder_us_dollars') or 0,
+                        tariff.get('year_one_coincident_peak_cost_us_dollars') or 0,
+                        tariff.get('year_one_bill_us_dollars') or 0
+                        ]))
+                    
+                    year_one_costs_bau = sum(filter(None, [
+                        tariff.get('year_one_energy_cost_bau_us_dollars') or 0,
+                        tariff.get('year_one_demand_cost_bau_us_dollars') or 0,
+                        tariff.get('year_one_fixed_cost_bau_us_dollars') or 0,
+                        tariff.get('year_one_min_charge_adder_bau_us_dollars') or 0,
+                        tariff.get('year_one_coincident_peak_cost_bau_us_dollars') or 0,
+                        tariff.get('year_one_bill_bau_us_dollars') or 0
+                        ]))
                     #Resilience Stats
-                    results['resilience_hours_min'] = resilience.get('resilience_hours_min', 'not evaluated')
-                    results['resilience_hours_max'] = resilience.get('resilience_hours_max', 'not evaluated')
-                    results['resilience_hours_avg'] = resilience.get('resilience_hours_avg', 'not evaluated')
+                    results['resilience_hours_min'] = resilience.get('resilience_hours_min') 
+                    results['resilience_hours_max'] = resilience.get('resilience_hours_max') 
+                    results['resilience_hours_avg'] = resilience.get('resilience_hours_avg') 
+
+                    if results['resilience_hours_max'] is None:
+                        results['resilience_hours_max'] = 'not evaluated'
+                    if results['resilience_hours_min'] is None:
+                        results['resilience_hours_min'] = 'not evaluated'
+                    if results['resilience_hours_avg'] is None:
+                        results['resilience_hours_avg'] = 'not evaluated'
+                    
+                    results['year_one_savings_us_dollars'] = year_one_costs_bau - year_one_costs
 
                     # PV Size
                     if pv is not None:
                         if pv['max_kw'] > 0:
                             results['pv_kw'] = pv.get('size_kw')
+                        else:
+                            results['pv_kw'] = 'not evaluated'
```

Comment:
@hdunham is there a specific reason you added the 'not evaluated'? I'm going to check with Andrew that the current UI can handle that

Reply:
@hdunham checked with Andrew and these should be null